#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

srslib_framework/MapStackMetadata metadata
srslib_framework/LogicalMap logical
srslib_framework/OccupancyMap occupancy